srcpkgs/neard/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'neard'
+pkgname=neard
+version=0.18
+revision=1
+build_style=gnu-configure
+configure_args="--disable-systemd --enable-tools"
+hostmakedepends="autoconf autoconf-archive automake libtool pkg-config"
+makedepends="glib-devel libnl-devel"
+short_desc="Near Field Communication (NFC) manager"
+maintainer="Jami Kettunen <jami.kettunen@protonmail.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/linux-nfc/neard"
+distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
```

Comment:
```suggestion
distfiles="https://github.com/linux-nfc/neard/archive/refs/tags/v${version}.tar.gz"
```

Reply:
If you don't mind I'll switch the URLs to https://git.kernel.org/pub/scm/network/nfc/neard.git; I'd use `${KERNEL_SITE}/network/nfc/neard-${version}.tar.xz` but they only have "pre-configured" archives up to v0.16 :/

Reply:
Ah nvm I just checked the README again, development moved away from kernel.org repo.